outbound-dialer/src/main/java/org/motechproject/nms/outbounddialer/domain/AuditRecord.java

```diff
@@ -0,0 +1,81 @@
+package org.motechproject.nms.outbounddialer.domain;
+
+import org.motechproject.mds.annotations.Entity;
+import org.motechproject.mds.annotations.Field;
+import org.motechproject.mds.annotations.UIDisplayable;
+
+@Entity(tableName = "nms_obd_audit_records")
+public class AuditRecord {
+    @Field
+    @UIDisplayable(position = 0)
+    private FileType type;
```

Comment:
Are any of these fields allowed to be null in the db?  If not should you add @Column(allowsNull = "false")